### DISCUSSION TITLE
Unknown file causing a problem

### DISCUSSION THREAD

**Discussion by sillypnk** [NvChad/NvChad#917] [Q&A]
Hi, i can't find the file that causing this problem:
![image](https://user-images.githubusercontent.com/102006493/160743602-822bb134-f549-4c02-8b43-b5e4a1650e23.png)

I already removed the custom folder and do `:PackerCompile` many times (even `:PackerSync`)
But still no luck.

**Comment by siduck** [comment]
whats the problem, i dont get it

**Reply by sillypnk** [reply]
That's what im asking, i don't even know what's the problem but it just changes my overrided theme and all in chadrc and all my plugins were disabled.

**Reply by sillypnk** [reply]
or should i say, chadrc is getting ignored

**Reply by sillypnk** [reply]
@siduck Why is the custom folder getting ignored? How do you even debug it

**Reply by siduck** [reply]
show your chadrc

**Reply by sillypnk** [reply]
> show your chadrc

https://srcb.in/3Q3mKNXh1q

**Reply by siduck** [reply]
btw you shouldnt copy whole of default_config.lua, only add the changes you want in chadrc

**Reply by sillypnk** [reply]
> btw you shouldnt copy whole of default_config.lua, only add the changes you want in chadrc

Okay, but did you figure out what's the problem..?

**Reply by sillypnk** [reply]
> > btw you shouldnt copy whole of default_config.lua, only add the changes you want in chadrc
> 
> Okay, but did you figure out what's the problem..?

@siduck Here's my structure
![image](https://user-images.githubusercontent.com/102006493/160781977-e6e3e029-8205-4445-b38b-3dc75ea3d0c2.png)

**Reply by siduck** [reply]
> > btw you shouldnt copy whole of default_config.lua, only add the changes you want in chadrc
> 
> Okay, but did you figure out what's the problem..?

it was hard as there are lots of default options/config in there so show me the one with changes only , like mine 

https://github.com/siduck/dotfiles/blob/master/nvchad/custom/chadrc.lua

**Reply by sillypnk** [reply]
> > > btw you shouldnt copy whole of default_config.lua, only add the changes you want in chadrc
> > 
> > 
> > Okay, but did you figure out what's the problem..?
> 
> it was hard as there are lots of default options/config in there so show me the one with changes only , like mine
> 
> https://github.com/siduck/dotfiles/blob/master/nvchad/custom/chadrc.lua

@siduck here, sorry for mentioning too much, im really losing patience rn. I really need to fix it
```lua
local M = {}

local cmp = require "cmp"

M.ui = {
   hl_override = "custom.highlights",
   theme = "gruvbox",
}

M.plugins = {
   status = {
      colorizer = true, -- color RGB, HEX, CSS, NAME color codes
      alpha = true, -- dashboard
   },
   options = {
      lspconfig = {
         setup_lspconf = "custom.plugins.lsp",
      },
   },
   default_plugin_config_replace = {
      nvim_tree = {
         view = {
            width = 30,
            hide_root_folder = false,
         },
         git = {
            enable = true,
         },
      },
      nvim_cmp = {
         mapping = {
            ["<C-,>"] = cmp.mapping.complete(),
         },
         sorting = {
            comparators = {
               cmp.config.compare.offset,
               cmp.config.compare.exact,
               cmp.config.compare.score,
               require("cmp-under-comparator").under,
               cmp.config.compare.kind,
               cmp.config.compare.sort_text,
               cmp.config.compare.length,
               cmp.config.compare.order,
            },
         },
      },

      bufferline = {
         options = {
            offsets = {
               {
                  filetype = "NvimTree",
                  text = "File Explorer",
                  highlight = "Directory",
                  text_align = "center",
                  padding = 1,
               },
            },
         },
      },
   },
   install = require "custom.plugins",
}

return M
```
